cpu/cc2538/include/periph_cpu.h

```diff
@@ -127,24 +127,15 @@ static inline uint8_t gpio_pp_num(gpio_t pin)
 }
 
 /**
- * @brief   Helper function to enable gpio hardware control
+ * @brief Alternative GPIO init function
```

Comment:
This seems a little arbitrary to me. I think this function initializes a pin in alternate function mode (or peripheral mode) or whatever TI is calling this. So not the function is an alternative to the default init function, but the resulting pin mode is an alternative to the default input or output mode...